languagetool-language-modules/el/src/test/java/org/languagetool/rules/el/GreekRedundancyRuleTest.java

```diff
@@ -0,0 +1,56 @@
+package org.languagetool.rules.el;
```

Comment:
Could you add the standard license header here, too?

Reply:
Done!